docs/data/flink_ml.yml

```diff
@@ -15,11 +15,24 @@
 # specific language governing permissions and limitations
 # under the License
 
+2.3:
+  name: "Apache Flink ML 2.3.0"
+  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-ml-2.3.0/flink-ml-2.3.0-src.tgz"
+  source_release_asc_url: "https://downloads.apache.org/flink/flink-ml-2.3.0/flink-ml-2.3.0-src.tgz.asc"
+  source_release_sha512_url: "https://downloads.apache.org/flink/flink-ml-2.3.0/flink-ml-2.3.0-src.tgz.sha512"
+  python_source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-ml-2.3.0/apache-flink-ml-2.3.0.tar.gz"
+  python_source_release_asc_url: "https://downloads.apache.org/flink/flink-ml-2.3.0/apache-flink-ml-2.3.0.tar.gz.asc"
+  python_source_release_sha512_url: "https://downloads.apache.org/flink/flink-ml-2.3.0/apache-flink-ml-2.3.0.tar.gz.sha512"
+  compatibility: ["1.17.*"]
```

Comment:
Should we mark it as 1.17.*, 1.16.*, 1.15.* as Flink-kuberneters-operator did?

Reply:
Currently Flink ML 2.3 python source only supports Flink 1.17. While Flink ML 2.3 Java source supports Flink 1.15 and Flink 1.6, users need to compile it with extra options (e.g. `mvn install -Pflink-1.16`) according to the Flink ML wiki [here](https://nightlies.apache.org/flink/flink-ml-docs-master/docs/development/build-and-install/#build-and-install-python-sdk).

Instead of specifying how to build Java source with the extra commands in the download page https://flink.apache.org/downloads/, it simpler to mention Flink 1.17 here. And I suppose users who want to use Flink ML 2.3 with Flink 1.15/1.16 find this information in the Flink ML wiki. What do you think?